system_launch/launch/system.launch.xml

```diff
@@ -45,5 +44,18 @@
       <arg name="config_file" value="$(find-pkg-share emergency_handler)/config/emergency_handler.param.yaml" />
     </include>
 
+    <!-- Dummy Diag Publisher -->
+    <group if="$(eval &quot;'$(var run_mode)'=='planning_simulation'&quot;)">
+      <include file="$(find-pkg-share system_launch)/launch/dummy_diag_publisher.launch.py">
+        <arg name="diag_config_path_array" value="
```

Comment:
Is it possible to replace this with `set_parameter`?

Reply:
I guess you can use this from Humble.
https://github.com/ros2/launch_ros/blob/d3970e5aa46b1449228f2d01d6b9e05555637d82/launch_ros/launch_ros/actions/set_parameters_from_file.py#L27-L28

Reply:
This also would be helpful.
https://github.com/autowarefoundation/autoware.universe/blob/main/common/global_parameter_loader/launch/global_params.launch.py

Reply:
It seems we should discuss more basic things first, so I'll resolve this now.